pyserini/pyclass.py

```diff
@@ -55,8 +55,10 @@
 ### Search
 
 JDocument = autoclass('org.apache.lucene.document.Document')
-JSearcher = autoclass('io.anserini.search.SimpleSearcher')
+JSimpleSearcher = autoclass('io.anserini.search.SimpleSearcher')
 JResult = autoclass('io.anserini.search.SimpleSearcher$Result')
```

Comment:
In the same way, `JSimpleSearcherResult` and `JSimpleNearestNeighborSearcherResult`? I know it's a bit long...
or `JSearchResult` or `JNearestNeighborResult`?

Reply:
sure, makes sense :+1: